excretor/src/env.rs

```diff
@@ -1,12 +1,16 @@
+use std::path::PathBuf;
+
 use clap::Parser;
 
-#[derive(Parser)]
+#[derive(Parser, Clone)]
 pub struct EnvVars {
     #[arg(env)]
     pub tummy_username: String,
     #[arg(env)]
     pub tummy_password: String,
     #[arg(env)]
+    pub static_assets_dir: PathBuf,
```

Comment:
Let's also set a default path here itself?

Reply:
I am not sure if that is possible. Let me check.

Reply:
Done.